src/components/whiteframe/demoDirectiveUsage/style.css

```diff
@@ -0,0 +1,45 @@
+div[md-whiteframe] {
```

Comment:
I don't know how i missed it, you DON'T want to restrict the use to use only div